### DISCUSSION TITLE
Replace `[]string` by `strings.StringBuilder`?

### DISCUSSION THREAD

**Discussion by lemeurherve** [updatecli/updatecli#487] [General]
... When it's used to concatenate strings

https://stackoverflow.com/questions/1760757/how-to-efficiently-concatenate-strings-in-go

WDYT?

**Comment by dduportal** [comment]
Good to know. Do you have any case in updatecli that you saw ? (Since slice of strings are not often used for concatenation)

**Comment by lemeurherve** [comment]
Yup, all the validationErrors in pkg/plugins/file/main.go for example

**Comment by dduportal** [comment]
Oh yeah good idea (i doubt that it makes a difference given that we are on a really newer version of go and the low volume of strings but could be awesome)
